docs/cli-contract.md

```
@@ -173,17 +178,23 @@ Auth subcommands:
-| `vm shell`, `vm attach` | Open an interactive shell for an existing VM. |
-| `vm rm`, `vm destroy`, `vm delete` | Destroy a VM. |
-| `vm ssh`, `vm ssh-info` | Print SSH connection info. |
-| `vm ssh-attach` | Internal attach helper. |
-| `vm exec` | Run a shell command inside a VM. |
+| `feed clear` | Clear persisted Feed workstream history. Supports `--yes` and `-y`. |
+
+VM and cloud subcommands:
+
+| Command | Contract |
+| --- | --- |
+| `vm|cloud ls`, `vm|cloud list` | List VMs. |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)